BitwardenShared/UI/Vault/Vault/VaultGroup/VaultGroupState.swift

```diff
@@ -63,14 +63,21 @@ struct VaultGroupState: Equatable, Sendable {
         // If the collection or trash are empty, return false.
         if case .collection = group {
             return false
-        } else if case .trash = group {
+        }
+        if case .sshKey = group {
+            return false
+        }
+        if case .trash = group {
             return false
         }
         return true
```

Comment:
🎨 Is it worth changing this into a switch statement now that there are multiple cases here?